src/component/turMgxComponent.js

```diff
@@ -0,0 +1,226 @@
+import '@oak-network/api-augment';
```

Comment:
The idea of code reuse is good, but the folder name component is not straight-forward.

Let’s make this file, `src/mangata/common.js` 

and move all mangata files to uder `src/mangata`

Please also create a README.md in that folder and copy past Mangata’s instructions from README.md to it.

Remember to update package.json.